src/main/scala/devices/spi/SPIPins.scala

```diff
@@ -5,15 +5,11 @@ import Chisel._
 import chisel3.experimental.{withClockAndReset}
 import sifive.blocks.devices.pinctrl.{PinCtrl, Pin}
 
-class SPISignals[T <: Data] (pingen: ()=> T, c: SPIParamsBase) extends SPIBundle(c) {
+class SPISignals[T <: Data](val pingen: ()=> T, c: SPIParamsBase) extends SPIBundle(c) {
```

Comment:
again why the inconsistency in private vs val vs not-private? What is the pattern here?